src/constants.ts

```diff
@@ -7,12 +7,16 @@ import SyncConfig from './domain/SyncConfig';
 export const CONFIG_FILENAME = 'sync-db.yml';
 export const CONNECTIONS_FILENAME = 'connections.sync-db.json';
 
+export const INJECTED_CONFIG_TABLE = '__injected_config';
```

Comment:
Append `sync_db` at the end? `__injected_config_sync_db`

Reply:
I think a prefix would be better - `__sync_db_injected_config` would be better.

Reply:
Sure.